data/mods/Magiclysm/Spells/magus.json

```diff
@@ -63,5 +63,107 @@
     "base_casting_time": 600,
     "energy_source": "MANA",
     "base_energy_cost": 350
+  },
+  {
+    "id": "magus_eldritch_blast",
+    "type": "SPELL",
+    "name": "Eldritch Blast",
+    "description": "A blast of concentrated magical power that obliterates a large area.",
+    "valid_targets": [ "hostile", "ground" ],
+    "effect": "projectile_attack",
+    "spell_class": "MAGUS",
+    "energy_source": "MANA",
+    "base_casting_time": 150,
+    "base_energy_cost": 200,
+    "difficulty": 5,
+    "max_level": 20,
+    "min_damage": 20,
+    "max_damage": 80,
+    "damage_increment": 3,
+    "damage_type": "none",
+    "min_aoe": 1,
+    "max_aoe": 5,
```

Comment:
this seems a little high

Reply:
In what regard? The damage or the aoe? 
it's a difficulty 5 spell and we so far don't have any spells that can significantly damage tougher enemies, much less one shot them and compete with higher end rifles or grenades. That is what this is intended to do, shore up that disparity.

Reply:
the aoe. if it maxed out at 3 ( diameter 6 ) i think the difficulty of 5 seems fine.

Reply:
You're right, I wasn't really thinking about how huge that is, 3 is plenty.